adoc/retail_chap_formulas.adoc

```diff
@@ -8,6 +8,8 @@ Formulas are pre-written Salt states, that are used to configure your {smr} inst
 
 This section lists the primary formulas shipped with {smr} and their configuration options.
 
+For working {smr} solution, all here presented formulas need to be properly configured. If you are not sure about correct configuration, use [command]``retail_branch_init`` command first. This command will create recommended formulas configuration which then can be manually edited.
```

Comment:
Try this:

`All the formulas in this section must be accurately configured for your {smr} installation to function correctly.
If you are unsure of the correct formula configuration details, run the [command]``retail_branch_init`` command before you begin to create the recommended formula configuration.
You can then manually edit the formulas as required.`

Reply:
Sure, updated.